wazo_call_logd/plugins/cdr/api.yml

```diff
@@ -437,6 +437,18 @@ definitions:
         - inbound
         - internal
         - outbound
+      remote_party:
+        type: object
+        properties:
+          kind:
+            type: string
+            enum:
+            - user
+            - conference
+            - meeting
+            - unknown
+          description:
```

Comment:
What is the key `description`? There is no need for another level of nesting, the `user_uuid` (and other) keys can come directly in the `remote_party` object.

Reply:
Fixed it